Tests/config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "api_key" : "",
-    "delivery_token" : "",
-    "environment" : ""
+    "api_key" : "blt477ba55f9a67bcdf",
+    "delivery_token" : "cs7731f03a2feef7713546fde5",
```

Comment:
Please avoid pushing sensitive information.

Reply:
Apologies, this has been addressed in new commit.